docker/js_ci/run.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+
+set -o errexit
+set -o nounset
+set -o verbose
+
+find /data/*.js* | xargs jshint
```

Comment:
Since this is a single command, can we do it directly in the Dockerfile rather than adding another shell script?